package.json

```diff
@@ -145,6 +148,10 @@
     "json5": "^2.2.3",
     "jsonschema": "1.5.0",
     "jwt-decode": "^3.1.2",
+    "klaro": "^0.7.18",
```

Comment:
Looks like you are also accidentally adding Klaro back in.  This has been replaced by Orejime on `main`.  So, this dependency can be removed.

Reply:
done